$OpenBSD: patch-Makefile,v 1.3 2013/03/17 18:35:26 ajacoutot Exp $

/usr/local/include/boost/exception/exception.hpp:158: warning:
'struct boost::exception_detail::error_info_container' has virtual functions but non-virtual destructor

--- Makefile.orig	Tue Mar 12 23:25:45 2013
+++ Makefile	Sun Mar 17 16:14:06 2013
@@ -22,7 +22,7 @@
 #                     found in PATH), this option has no effect.
 #
 
-OPTIMIZE=yes
+OPTIMIZE=no
 CCACHE?=ccache
 USE_CCACHE?=$(shell which $(CCACHE) 2>&1 > /dev/null && echo yes)
 ifneq ($(USE_CCACHE),yes)
@@ -34,7 +34,7 @@ BASE_CXXFLAGS += -O2
 endif
 
 # Initial compiler options, used before CXXFLAGS and CPPFLAGS.
-BASE_CXXFLAGS += -g -fno-inline-functions -fthreadsafe-statics -Wnon-virtual-dtor -Werror -Wignored-qualifiers -Wformat -Wswitch
+BASE_CXXFLAGS += -fno-inline-functions -fthreadsafe-statics -Wnon-virtual-dtor -Werror -Wno-non-virtual-dtor -Wformat -Wswitch
 
 # Compiler include options, used after CXXFLAGS and CPPFLAGS.
 INC := $(shell pkg-config --cflags x11 sdl glu glew SDL_image libpng zlib)
